src/controllers/__tests__/ticket.test.ts

```diff
@@ -3,49 +3,152 @@ import { build } from "../../../tests/helper";
 const app = build();
```

Comment:
I think we don't need to connect to DB to test controllers. Let's discuss later it.